terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -658,6 +662,21 @@ if ${debug}; then set -x ; fi
 # This way you can catch the error in case mysqldump fails in `mysqldump |gzip`, for example.
 set -o pipefail
 
+# Convert to lowercase for case insensitive check
+secure_mode=$(echo "$SECURE_MODE" | tr '[:upper:]' '[:lower:]')
+
+# Check if script needs  to be validated for secure production mode
+if [ "$secure_mode" = "true" ]; then
```

Comment:
It seems like this switch between IDCS port and managed server port would be useful for non-production mode as well. Seems like an oversight in the existing script to me. @Mahuwa-Barman , do you agree?

Reply:
Yes